server/slack/webhooks.js

```diff
@@ -0,0 +1,76 @@
+import { HTTP } from "meteor/http";
+
+const SLACK_USER_URL = "https://slack.com/api/users.info";
+
+const webhooks = {
+  handleNewEvent(params, req, res) {
+    console.log("webhooks.handleNewEvent", req.headers);
+    const { token, type, challenge, event } = req.body || {};
+    const { slackAppToken } = Meteor.settings;
+
+    if (!token || token !== slackAppToken) {
+      res.statusCode = 404;
+      res.end();
+      return;
+    }
+
+    if (type === "url_verification" && challenge) {
+      res.statusCode = 200;
+      res.write(challenge);
+      res.end();
+      return;
+    }
+
+    if (type === "event_callback" && event) {
+      res.statusCode = 200;
+      res.end();
+      webhooks.processEvent(event);
+      return;
+    }
+    res.statusCode = 404;
+    res.end();
+  },
+
+  // First get User and it's email and it's timezone from slack API
+  // Find Meteor user whos email is same as slack's one
+  // If Meteor user's and slack user's have same email then create hangout
+  processEvent(event) {
+    console.log("webhook.processEvent", event);
+    const { user: slackUserId, text, type, channel, ts } = event || {};
+    if (type !== "message") return;
```

Comment:
eslint should throw here ping @distalx

Reply:
@sebbel Do we use any style guide for codebuddies? I can't find any eslint config.

Reply:
@sebbel What do you mean by eslint throwing here? Throwing an error?

@gauravchl We have https://docs.codebuddies.org/code-contribution/coding_style, and Prettier does automatically updates the code in the PR to be a certain style. Let me know if you need help with any of the remaining tasks!

Reply:
@lpatmo Thanks Linda! I will tackle remaining tasks this weekend, i might need some help with setting up slack app on production env.

Reply:
@gauravchl Thanks, sounds great! I will be working on CB too this weekend. Re: hosting -- were you thinking something like a DigitalOcean droplet? We should have funds still in our DigitalOcean account from the DO sponsorship. That's where we're hosting https://github.com/codebuddies/greetbot at the moment. (Not sure if it's possible to build _on top_ of greetbot for the purposes of this integration.)

Reply:
Ah, that makes sense. I can def. add keys to both settings-development.json (so we can test it out) and then to settings-production.json for the release.